Resources/doc/cache-resolver/web_path.rst

```diff
@@ -4,8 +4,26 @@
 Web Path Resolver
 =================
 
-The ``WebPathResolver`` resolver enabled cache resolution using the
-web path of your application.
+This cache resolver (``WebPathResolver``) enables cache resolution for
+local, web-path-based setups. This means images will be cached on your
+local filesystem, within the web path of your Symfony application.
+
+The resulting web path is computed by taking a number of factors into
+account, including the `request context`_, as provided by the Symfony
+HTTP kernel.
+
+.. tip::
+
+    The `request context`_ is most notably used to determine the HTTP
+    scheme used for the final URL. If you use a proxy to offload TLS
+    traffic decryption and need the resolver to generate secure URLs,
+    you will need to appropriately configure Symfony's `trusted proxies`_.
+    If you utilize `embedded controllers`_ in your templates, you must
+    add ``localhost`` to your trusted proxy configuration.
```

Comment:
Not so sure about that one… trusted proxies configuration maybe? or "add `localhost` as a trusted proxy"?